client/app/bundles/course/assessment/submission/pages/SubmissionEditIndex/SubmissionEditForm.jsx

```diff
@@ -401,7 +402,13 @@ const SubmissionEditForm = (props) => {
     const editable = !attempting && graderView;
     const visible = editable || published;
 
-    return visible ? <QuestionGrade editable={editable} id={id} /> : null;
+    return visible ? (
+      <QuestionGrade
+        editable={editable}
+        handleSaveGrade={() => handleSaveGrade(id)}
```

Comment:
Since `id` is also passed in to this component, it is better to just pass the `handleSaveGrade` directly and when the function is invoked inside `QuestionGrade`, you can pass the `id`. Also, this reduces the need to recreate a new function.